printc/__init__.py

```diff
@@ -1,17 +1,21 @@
 from sys import stdout
 
+    
+class Color:
+    def __init__(self, c=15, flush=False):
```

Comment:
How about meaningful variable names? Say `color_number` instead of `c` or something else that is more verbose than singe letter (That goes to me as well as I named this var ;))